docs/pages/reference/cli.mdx

```diff
@@ -159,10 +159,42 @@ $ tsh help
 
 ### tsh version
 
-Print the version of your `tsh` binary:
+Print the version of your `tsh` binary and current profile Teleport proxy version:
+
+```code
+$ tsh version [<flags>]
+```
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `-f, --format`  | `text`  | text, json, yaml | Format for version output |
+| `--client-only` | none | none | Only display the `tsh` binary version and do not retrieve the proxy version. |
+
+#### Examples
 
 ```code
 $ tsh version
+Teleport v(=teleport.version=) git: go(=teleport.golang=)
+Proxy version: (=teleport.version=)
+```
+
+Display in JSON format
+```code
+$ tsh version --format=json
+```
+```json
+{
+  "version": "(=teleport.version=)",
+  "gitref": "",
+  "runtime": "go(=teleport.golang=)",
+  "proxyVersion": "(=teleport.version=)"
```

Comment:
If we're showing the proxy version we should probably also include the hostname of the proxy, otherwise it's not totally clear which proxy this version applies to.

Reply:
hostname added